jobs/validate/calico-spec

```
@@ -43,7 +43,8 @@ function juju::bootstrap
 
       ret=$?
       if (( ret > 0 )); then
-          juju::deploy-failure $ret
+          # Fail Deploy Early
+          juju::deploy-report $ret
```

a special case to fail fast on calico deployments